Source/Core/Core/PowerPC/JitCommon/JitCache.cpp

```diff
@@ -485,7 +488,10 @@ JitBlock* JitBaseBlockCache::MoveBlockIntoFastCache(u32 addr, CPUEmuFeatureFlags
   // And create a new one
   size_t index = FastLookupIndexForAddress(addr, feature_flags);
   if (m_entry_points_ptr)
+  {
+    m_entry_points_arena.EnsureMemoryPageWritable(index * sizeof(u8*));
     m_entry_points_ptr[index] = block->normalEntry;
+  }
   else
     m_fast_block_map_fallback[index] = block;
```

Comment:
You probably want to add curlies here as well for balance.
```suggestion
  else
  {
    m_fast_block_map_fallback[index] = block;
  }
```